### PR TITLE
add theoplayer docs

### PR DESCRIPTION
**Pull request by jsanford8**
this is almost ready - just one thing left to do on the videochange piece, waiting on theoplayer to get back to me

### CODE REVIEW

**Comment by jsanford8** [comment]
this is ready now

File: theoplayer/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Include the theoplayer-mux file after the core THEOplayer javascript file in the page. Use the Mux-hosted version of the script to receive automatic updates (the API will not change within major versions).
```
Comment (theRealWardo) [inline theoplayer/getting-started.md:6]: missing `Sort` and `Title` metadata?

File: theoplayer/options-and-metadata.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Title: Options and Metadata
+Sort: 2
+*/
+
+Name	| Description	| Default
+----------- | ----------- | --------
+debug	| Put the SDK in debug mode to log operational details	| false
+data | User, page, player, and video metadata for the video | { }
+
+In the Objective-C SDKs, *options are provided via the MUXSDKCustomerPlayerData and MUXSDKCustomerVideoData objects*.
```
Comment (theRealWardo) [inline theoplayer/options-and-metadata.md]: remove as this is now language/player specific

File: theoplayer/options-and-metadata.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+- Video details (prepended by `video_`) describe the current video that's playing and are all reset automatically when [changing the video](changing-the-video). This metadata would come from your internal CMS or video management system.
+- Player details (prepended by `player_`) describe the player configuration that's being used and should be set whenever monitoring is started on a new player. They do not reset when the video is changed.
+- All other details will persist until explicitly changed.
+
+In the Objective-C SDKs, *names are converted to lowerCamelCase setters and getters per Apple's naming guidelines*. See the <a href="https://github.com/muxinc/stats-sdk-objc/blob/master/Frameworks/iOS/release/MUXSDKStats.framework/Headers/MUXSDKCustomerPlayerData.h" target="_blank">MUXSDKCustomerPlayerData.h</a> and <a href="https://github.com/muxinc/stats-sdk-objc/blob/master/Frameworks/iOS/release/MUXSDKStats.framework/Headers/MUXSDKCustomerVideoData.h" target="_blank">MUXSDKCustomerVideoData.h</a> header files for a complete list of names.
```
Comment (theRealWardo) [inline theoplayer/options-and-metadata.md]: remove as this is now language/player specific

**Review by theRealWardo — APPROVED** [review]
few things, otherwise looks good

File: theoplayer/getting-started.md (hunk 1 above)
Comment (jsanford8) [reply theoplayer/getting-started.md:6]: Huh, all of them are. I'll add it to all of them

File: theoplayer/options-and-metadata.md (hunk 2 above)
Comment (jsanford8) [reply theoplayer/options-and-metadata.md]: Same with all web ones... removed

**Pull request merged**